src/components/SignUp/Landing.tsx

```diff
@@ -61,22 +56,26 @@ const Landing: React.FC<{
         lg="5"
         onClick={setFormCheck}
       >
-        <Image alt="" src={blob} />
-        <div>
-          <p>{text}</p>
-          <div>{input}</div>
-        </div>
+        <Image alt="" src={icon} />
+
+        <div>{text}</div>
```

Comment:
We'll want to use semantic text tags for pieces of text (notice this was a `p` before).